ezCMakeConfig.cmake

```diff
@@ -13,8 +13,8 @@ set_property(GLOBAL PROPERTY EZ_CMAKE_RELPATH_CODE "Code")
 
 set_property(GLOBAL PROPERTY EZ_CONFIG_PATH_7ZA "${CMAKE_SOURCE_DIR}/Data/Tools/Precompiled/7za.exe")
 
-set_property(GLOBAL PROPERTY EZ_CONFIG_QT_WINX64_VERSION "Qt-5.13.0-vs141-x64")
-set_property(GLOBAL PROPERTY EZ_CONFIG_QT_WINX64_URL "https://github.com/ezEngine/thirdparty/releases/download/Qt-5.13.0-vs141-x64/Qt-5.13.0-vs141-x64.zip")
+set_property(GLOBAL PROPERTY EZ_CONFIG_QT_WINX64_VERSION "Qt6-6.4.0-vs143-x64")
+set_property(GLOBAL PROPERTY EZ_CONFIG_QT_WINX64_URL "http://stuff.benjamin-thaut.de/ez/Qt6-6.4.0-vs143-x64.7z")
```

Comment:
This is a temporary URL that has to be replaced with a github hosted package instead.